UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSEndToEndScenarioWithInvalidProductIdentifier.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSEndToEndScenarioWithInvalidProductIdentifier")]
+    public class POSEndToEndScenarioWithInvalidProductIdentifier
+    {
+        private string fssJwtToken;
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
+
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            AuthTokenProvider authTokenProvider = new();
+            fssJwtToken = await authTokenProvider.GetFssToken();
+
+            HttpResponseMessage apiResponse = MockHelper.ConfigureFM(posWebJob.MockApiBaseUrl, posWebJob.FMConfigurationInValidProductIdentifier);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+            await CommonHelper.RunWebJob();
+        }
+
+        [Test]
+        public async Task WhenICallFileDownloadEndpointWithInvalidProductIdentifier_ThenABatchWithErrorTxtIsGenerated()
+        {
+            HttpResponseMessage responseMessage = await FssBatchHelper.VerifyErrorTxtExist(fssJwtToken);
+            responseMessage.StatusCode.Should().Be((HttpStatusCode)200);
```

Comment:
is this not the same as the previous two tests but the invalid case?

Reply:
As discussed with Jason, we have a reason to keep all scenarions as separate files. For each test case scenario, we run the webjob and everytime stub creates new batches according to given scenario.

Category attribute is added to run all test case files sequentially instead of parallel. Otherwise, we get 409-conflict from webjob.